src/App.tsx

```diff
@@ -1,11 +1,17 @@
 import React from 'react'
 import DemoButton from './DemoButton'
 import './styles/index.scss'
+import Icon from './atoms/Icon'
+import { Styles } from './constants/enum'
 
 const App: React.FC = () => {
   return (
     <div>
       <DemoButton />
+      <Icon
```

Comment:
It could have been cool to have en "demo component" to just show multiple things like I did for the buttons. It will be removed at the end anyway so we can do what we want with the App component :D

Reply:
Done.